src/app/lib/config/services/gahuza.ts

```diff
@@ -82,6 +82,7 @@ export const service: DefaultServiceConfig = {
           'Ibindi vyerekeye ingene urwo rubuga rugutwara uburyo (ama mega) buke',
         informationPageLink: 'https://www.bbc.com/gahuza/articles/cn7y7pvem0vo',
         dataSaving: 'Ahagusaba uburyo (ama mega) buke',
+        dataSavingVersion: 'Nyandiko gusa',
```

Comment:
What will be the use cases for these 2 translations? Is one for the Lite Site banner and the other for the link below the Article header?

Reply:
Yes - I just did a google translate, and this means "text only" whereas the message above it (Ahagusaba uburyo (ama mega) buke) is: "It requires a few (mega) resources."

Reply:
Ah ok thats grand.

Reply:
Think the naming of the translation is ok. Was thinking of something more specific, like `articleDataSavingLinkText`, but I guess if we want to reuse it elsewhere it may need something more generic.

Reply:
and `'Nyandiko gusa'` is `Text only`